### ISSUE TITLE
Protege should support extensible internationalisation

### ISSUE THREAD

**Issue by matthewhorridge** [protegeproject/protege#694]
As the title suggests.  At the moment this has to be done by changing the plugin.xml file, which is less than ideal.

**Comment by matthewhorridge** [comment]
Most of the strings used in the UI (menus etc.) are specified in the plugin.xml.  It would be good if we had resource bundles that could be automatically switched depending on locale settings.  We now do this in WebProtege (https://webprotege.stanford.edu) for languages where we have translations  - we currently have English, Portuguese, Slovenian and partial German - we need a fluent German speaker to do a better complete German translation ;)

**Comment by matthewhorridge** [comment]
p.s. [see here](https://github.com/protegeproject/webprotege/tree/master/src/main/resources/edu/stanford/bmir/protege/web/client) - this is how languages are specified in WebProtege

**Comment by ykazakov** [comment]
OK, I see some menu strings (names and tooltips) in plugin.xml, but most other strings used in the UI are hardcoded. For example, the names of the checkboxes in Displayed Inferences Reasoner preferences. So changing plugin.xml would not be enough.

**Comment by matthewhorridge** [comment]
That's right.  Changing the plugin.xml wouldn't be a good idea anyway.  Things need to be refactored to use resource bundles.
